stage #2.5/module #4 REST Basics - Spring Web/Spring Rest/content.md

```diff
@@ -0,0 +1,23 @@
+## REST with Spring
+
+### Understanding REST in Spring
+
+The Spring framework supports two ways of creating RESTful services:
+- using MVC with ModelAndView;
+- using HTTP message converters.<br>
+
+The **ModelAndView approach** is older and much better documented, but also more verbose and configuration heavy. 
+The new approach, based on **HttpMessageConverter and annotations**, is much more lightweight and easy to implement.
+Configuration is minimal, and it provides sensible defaults for what you would expect from a RESTful service.
+
+Let's consider how to build REST application using Spring Web framework step by step.
+
+### [REST Controllers in Spring](./spring-rest-controller-content.md)
```

Comment:
I ask you to put this sub-chapters directly in content.md, not via links.
Since in autocode it will be one file, it will much easier you put content in one file